webshopping/lolshopping/templates/design/home.html

```diff
@@ -52,9 +52,10 @@
         <div class="col-lg-3 abc">
             <img class ="thumbnail" src="{{champs.imageURL}}">
             <div class="box-element product">
-                <div class="namez">
-                <h6>{{champs.name}}</h6></div>
-                <h6>{{champs.role}}</h6>
+            <span class="namez">
+                <h6>{{champs.name}}</h6></span>
+            <span class= "rolez">
```

Comment:
Same thing here, why does class name have "z" in it.

Reply:
same reason as before, but will change